release-0.0.6-asc.yaml

```diff
@@ -0,0 +1,417 @@
+apiVersion: v1
```

Comment:
Was this made a part of the PR accidentally?

Reply:
yes